doc/CMakeLists.txt

```diff
@@ -1,76 +1,99 @@
-#
 # Copyright (c) 2019 Nordic Semiconductor ASA
-#
 # SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+
+# Builds combined documentation for nRF, Zephyr, MCUboot, and nrfxlib.
+#
+# We use our own Sphinx configuration files when building the documentation set
+# for each repository, instead of reusing configuration files. See e.g.
+# doc/nrf/conf.py and doc/zephyr/conf.py.
 #
+# Intersphinx (http://www.sphinx-doc.org/en/master/usage/extensions/intersphinx.html)
+# is used to link documentation sets together. It is configured in the Sphinx
+# configuration files.
+#
```

Comment:
You could add Doxygen here as well (it is run before the Sphinx build).

Reply:
Added. It's only for nRF, right?

Reply:
No, it's for nrf, nrfxlib, and zephyr.

Reply:
I only see

```
add_custom_target(
  nrf-doxy
  COMMAND ${CMAKE_COMMAND}
    -DCOMMAND=${DOXYGEN_EXECUTABLE}
    -DARGS="${NRF_DOXYFILE_OUT}"
    -DOUTPUT_FILE=${NRF_DOXY_LOG}
    -DERROR_FILE=${NRF_DOXY_LOG}
    -DWORKING_DIRECTORY=${CMAKE_CURRENT_LIST_DIR}
    -P ${ZEPHYR_BASE}/cmake/util/execute_process.cmake
)
```

and (in `doc/nrf/nrf.doxyfile.in`)

```
INPUT                  = @NRF_BASE@/applications \
                         @NRF_BASE@/lib \
                         @NRF_BASE@/include
```

Wonder how it ends up generating the other ones.

Reply:
Line 297 for nrfxlib, and the Zephyr one should be in Zephyr's CMakeLists.txt.

Reply:
Right... derp. Got confused while searching and thought the `nrf-doxy` and `nrfxlib-doxy` target definitions were the same code. :)